shepherd

```diff
@@ -72,10 +75,16 @@ main() {
     echo "Connection to insecure registry available"
   fi
 
+  supports_no_resolve_image=false
+  if [[ ${WITH_NO_RESOLVE_IMAGE+x} ]]; then
+    supports_no_resolve_image=true
+    echo "Deployment with no resolve image allowed"
```

Comment:
`without resolving image`

Reply:
I have change the information message as requested